api/routes.go

```diff
@@ -72,6 +72,8 @@ const (
 	organizationDeleteParticipantsEndpoint = "/organizations/{address}/participants"
 	// POST/PUT/GET/DELETE /organizations/{address}/meta to add/set/get/delete the organization metadata
 	organizationMetaEndpoint = "/organizations/{address}/meta"
+	// POST /organizations/{address}/ticket to create a new ticket to our support system
+	organizationCreateTicketEndpoint = "/organizations/{address}/tickets"
```

Comment:
`POST /organizations/{address}/ticket` (no plural) sounds more reasonable to me (unless there are plans to implement a `GET /organizations/{address}/tickets` but i don't think that's the case, right?

Reply:
in fact the comment on top says `ticket`